blocks/identity-block/components/FormInputField/index.jsx

```diff
@@ -25,34 +25,35 @@ const FormInputField = ({
   validationPattern,
 }) => {
   const [valid, setValid] = useState(true);
+  const [value, setValue] = useState(defaultValue);
```

Comment:
wait, aren't we tracking state in the parent? this is new

Reply:
yeah, I needed to hold that value to pass it along to the useEffect so that a change will trigger a revalidation.  I really didn't want to hold this value internally, but I don't see a way around it.